4.0/entrypoint.sh

```diff
@@ -47,4 +47,6 @@ psql $POSTGRES_DB -h $POSTGRES_HOST -p $POSTGRES_PORT  -U $POSTGRES_USER -c "UPD
 psql $POSTGRES_DB -h $POSTGRES_HOST -p $POSTGRES_PORT  -U $POSTGRES_USER -c "UPDATE vdc_options set option_value = '$HOST_INSTALL' where option_name = 'InstallVds';"
 psql $POSTGRES_DB -h $POSTGRES_HOST -p $POSTGRES_PORT  -U $POSTGRES_USER -c "UPDATE vdc_options set option_value = '$HOST_USE_IDENTIFIER' WHERE option_name = 'UseHostNameIdentifier';"
 
+engine-config -s BlockMigrationOnSwapUsagePercentage=99999
```

Comment:
We should also consider doing something like suggested here https://github.com/rmohr/docker-ovirt-engine/issues/16, to not have to create dozens of environment variables